apps/judicial-system/web/src/routes/Shared/Cases/cases.graphql

```diff
@@ -34,6 +34,10 @@ query Cases {
     initialRulingDate
     rulingDate
     rulingSignatureDate
+    court {
```

Comment:
Is this also needed in other case list queries, such as for the prison admin or defenders?

Reply:
Not currently. @thorhildurt is however working on adding this feature for prisons so this needs to be addressed there.